src/validation.cpp

```diff
@@ -5269,6 +5270,34 @@ CChainState& ChainstateManager::InitializeChainstate(const uint256& snapshot_blo
     return *to_modify;
 }
 
+/**
+ * Return the expected assumeutxo value for a given height, if one exists.
+ *
+ * @param height[in]         Get the assumeutxo value for this height.
+ * @param expected_out[out]  Set to the expected assumeutxo hash value if one exists.
+ *
+ * @returns bool - false if no assumeutxo value exists for the given height.
+ */
+static bool ExpectedAssumeutxo(int height, uint256& expected_out)
+{
+    const CChainParams& params = ::Params();
```

Comment:
in commit d6396c4be4

For pure-utility functions it would be good to pass in `params` instead of relying on the global

Reply:
Done